dynawo/sources/API/CRT/test/res/criteria.crt

```diff
@@ -14,7 +14,7 @@
 <criteria xmlns="http://www.rte-france.com/dynawo">
   <busCriteria>
     <parameters id ="busCritId" scope="DYNAMIC" type="LOCAL_VALUE" uMaxPu="0.8" uNomMin="225"/>
-    <component id="MyId"/>
+    <component voltageLevelId="MyVoltageLevelId" id="MyId"/>
```

Comment:
I don't see the associated test in the unit test?

Reply:
Add missing test in `testXmlFileImporter`